playwright-action/helpers/playwrightUtils.js

```diff
@@ -0,0 +1,47 @@
+//@ts-check
+
+//Defining test functions for playwright to be used in the spec file
+
+//Login to be used for settingup tests
+export async function login(page, username, password) {
+      await page.goto('https://meshery.layer5.io/login');
+      await page.locator('input[name="identifier"]').fill(username);
+      // Press the Tab key (to move to the password field)
+      await page.locator('input[name="identifier"]').press('Tab');
+      await page.fill('input[name="password"]',password)
+      await page.waitForSelector('input[name="password"]');
+      await page.fill('input[name="password"]', password);
+
+      // Wait for the "Sign in" button to appear and then click it
+      const signInButton = await page.waitForSelector('button[type="submit"]');
+      await signInButton.click(); 
+      // Wait for some time (e.g., 5 seconds) for any post-login actions or page navigation to complete
+      await page.waitForTimeout(5000);
```

Comment:
@Philip-21 `Never wait for timeout in production. Tests that wait for time are inherently flaky. Use [Locator](https://playwright.dev/docs/api/class-locator) actions and web assertions that wait automatically.`

For reference, see https://playwright.dev/docs/api/class-page#page-wait-for-timeout

Reply:
A solid observation. 🙌